tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs

```diff
@@ -44,7 +56,7 @@ public async Task ThenOnGetAsync_LocalOfferDetailWithReferralNotEnabled()
         localOfferDetailModel.PageContext.HttpContext = httpContext;
 
         //Act 
-        var result = await localOfferDetailModel.OnGetAsync("NewId", serviceDto.Id) as PageResult;
+        var result = await localOfferDetailModel.OnGetAsync("NewId", (serviceDto != null) ? serviceDto.Id : string.Empty) as PageResult;
 
         //Assert
```

Comment:
could you please add an assert Website.should.be.equalto(url)?

Reply:
Does not exactly work like that but changed anyway